demos/TransferLearning/README.md

```diff
@@ -0,0 +1,60 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
```

Comment:
I don't think your `README` is formatted the way you expect it to be in GitHub.

See https://github.com/plaidml/plaidml/blob/1b10882100a826b8a64a8d97ed3ec7000a6abf0f/demos/TransferLearning/README.md